OpenRA.Mods.Common/Traits/Carryall.cs

```diff
@@ -114,6 +114,11 @@ public enum CarryallState
 		/// <summary>Offset between the carryall's and the carried actor's CenterPositions</summary>
 		public WVec CarryableOffset { get; private set; }
 
+		public bool IsCargoValid(Actor cargo, Carryable carryable)
```

Comment:
This should be renamed into `CanPickUp`.

Reply:
Renamed to `IsPickupValid`. 

`CanPickUp` would suggest more checks about the carryall itself

Reply:
`CanPickUp(cargo)` is almost a sentence describing what it does, but ok.